test/sqllogictest/attributes/mir_column_types.slt

```diff
@@ -9,7 +9,7 @@
 
 statement ok
 CREATE TABLE t (
-  a int,
+  a iasdnt,
```

Comment:
I guess we don't want to merge this 😀 .

Reply:
Yep, the commit has a DNM and this would fail CI anyway, thus blocking the merge, will remove after CI verification.